packages/dev/core/src/Materials/Textures/renderTargetTexture.ts

```diff
@@ -457,7 +461,8 @@ export class RenderTargetTexture extends Texture implements IRenderTargetTexture
         samples?: number,
         creationFlags?: number,
         noColorAttachment?: boolean,
-        useSRGBBuffer?: boolean
+        useSRGBBuffer?: boolean,
+        gammaSpace?: boolean

```

Comment:
I think that now that we have a `RenderTargetTextureOptions` object parameter, we shouldn't add new parameters in this way (that's what we did for the last option - `colorAttachment` - that we added)?

Reply:
yup totally agree I ll keep only the option ones.